router/http_test.go

```diff
@@ -514,7 +527,7 @@ func (s *S) TestNoBackends(c *C) {
 	c.Assert(res.StatusCode, Equals, 503)
 	data, err := ioutil.ReadAll(res.Body)
 	c.Assert(err, IsNil)
-	c.Assert(string(data), Equals, "Service Unavailable\n")
+	c.Assert(string(data), Equals, http.StatusText(http.StatusServiceUnavailable))
```

Comment:
It was intentional that I didn't use the constants in the tests. I felt it was a better guarantee against regressions, though that might not really be true.

Regardless, it looks like you've lost the curl-friendly `\n` that was in these responses before.